internal/v1/handler_blueprints_test.go

```diff
@@ -295,6 +299,14 @@ func TestHandlers_GetBlueprintComposes(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, blueprintId, *result.Data[0].BlueprintId)
 	require.Equal(t, 2, *result.Data[0].BlueprintVersion)
+
+	//// get composes for non-existing blueprint
+	respStatusCode, body = tutils.GetResponseBody(t, fmt.Sprintf("http://localhost:8086/api/image-builder/v1/experimental/blueprints/%s/composes?blueprint_version=1", uuid.New().String()), &tutils.AuthString0)
+	require.Equal(t, 200, respStatusCode)
+	err = json.Unmarshal([]byte(body), &result)
+	require.NoError(t, err)
+	require.Equal(t, 0, len(result.Data))
+	require.Equal(t, 0, result.Meta.Count)
```

Comment:
I'm unsure here: should we rather return 404 as well, this feels bit weird to have empty response for non-existent blueprint

Reply:
Yes, this is interesting; we'd have to distinguish between Blueprint Not Found vs No Composes. This is the case for most of the endpoints where we use JOINS. I am happy to discuss how much we want to enhance this.

Reply:
If a bluprint does not exist in a `/blueprints/XYZ/whatever` path to me it feels logical to see Not Found just like this was a directory on a file system.